### DISCUSSION TITLE
Automate video caputring from meetings.

### DISCUSSION THREAD

**Discussion by g1ronn1mo** [chhoumann/quickadd#578] [Q&A]
Introduktion: Videos can be captured with the xbox capture feature. To activate it, use the keys `windows` + `g`. These Videos can be processed with  the obsidian plugin [transcription](https://github.com/djmango/obsidian-transcription) into text. The wanted behavior is that videos are automatically stored inside the vault and the link automatically copied to the clipboard. 

To automate this, follow the steps: 
	- [x] Copy Xbox Capture Folder inside the Media Folder 
	- [ ] Hotkey for a command. 
	- [ ] The command should insert the last added file for a specific folder into the current open file and maybe run the transcribe command 

Is this maybe possible with quickadd?

**Comment by Jawabiscuit** [comment]
QA enthusiast here, this is very achievable. What I do is use a macro to call a template to 1) create a new file with generated content and 2) call the transcription command. It's entirely automatable with a few simple steps in QA + transcription however I use templater for extra features when creating new files from templates. What I need now is for either the transcription plugin or a tool of my own to download videos that I want summarized from the internet, the former would be nice for a seamless experience. This is probably wading into copyright territory though so it's probably up to me to do that part. Good luck, and lemme know if I can break it down more for you!

**Reply by g1ronn1mo** [reply]
Thank you, would be great to have a test vault. You can upload the vault to Github and share it here.
